src/test/cypress/integration/ModelingExercise.spec.ts

```diff
@@ -3,7 +3,17 @@ import { artemis } from '../support/ArtemisTesting';
 
 // https://day.js.org/docs is a tool for date/time
 import dayjs from 'dayjs';
-
+import { CourseManagementPage } from '../support/pageobjects/CourseManagementPage';
+import { CreateModelingExercisePage } from '../support/pageobjects/CreateModelingExercisePage';
+import { ModelingExerciseExampleSubmissionPage } from '../support/pageobjects/ModelingExerciseExampleSubmissionPage';
+
+// pageobjects
+const courseManagement: CourseManagementPage = artemis.pageobjects.courseManagement;
+const createModelingExercise: CreateModelingExercisePage = artemis.pageobjects.createModelingExercise;
+const modelingExerciseExampleSubission: ModelingExerciseExampleSubmissionPage = artemis.pageobjects.modelingExerciseExampleSubmission;
```

Comment:
Pageobject types should be aquired automatically, so you can remove this
```suggestion
const courseManagement = artemis.pageobjects.courseManagement;
const createModelingExercise = artemis.pageobjects.createModelingExercise;
const modelingExerciseExampleSubission = artemis.pageobjects.modelingExerciseExampleSubmission;
```